release/models/wifi/README.md

```diff
@@ -0,0 +1,34 @@
+# wifi-models
+Development of configuration and operational state models for WiFi
+
+Open Config is a collaboration between network operators to develop and standardize a set of vendor-neutral network configuration models. Models are written using the YANG data modeling language (IETF RFC 6020).
+
+The configuration models in this repository are made available under the Apache 2.0 license (see the LICENSE file). Since the models are intended to be published in the IETF, participants must be willing to adhere to the IETF Note Well statement as well as BCP 78 and BCP 79.
```

Comment:
do we plan to publish the wifi models in IETF?  if not sure, you can add these statements if/when we do that.

Reply:
I'd say not sure. Removed IETF references from README.